src/app/team/serviceForm/membershipCriteria/options.js

```diff
@@ -4,12 +4,7 @@ const options = [
     values: [true, false],
   },
   {
-    label: 'Referal Letter',
```

Comment:
The commit that adds this to the Proofs page will be in the other PR, I assume?

Reply:
@abimaelmartell I didn't see the "Referral Letter" option in the proofs PR.
Shouldn't that be added to the proofs page, since it was removed from here?